common/errors.go

```diff
@@ -0,0 +1,39 @@
+package common
```

Comment:
Should this just go in the `api` package?

Reply:
Good point! The only question is `api` is currently all protobuf stuff, it otherwise fits better in `api` since these are all around APIs. I'll move it over unless other folks have different thought/suggestion.